### ISSUE TITLE
Cloudflared Autoupdate this morning has broken all websites as it is changing the request URL

### ISSUE THREAD

**Issue by Routhinator** [cloudflare/cloudflared#137]
Cloudflared auto-updated on our systems this morning and took down all of our sites because it is now prepending the local proxy and port it is forwarding the request to to the URL, causing our webservers to no longer recognize the request.

Pre update:
```
POST /this/url/should/be/relative
```

Post update:
```
POST http://127.0.0.1:8890/this/url/should/be/relative
```

**Comment by sssilver** [comment]
The breaking change has been rolled back.

**Comment by darthShadow** [comment]
Sorry for bumping this but is this something on the cloudflare server-side or will it require us to manually roll-back cloudflared to the older version?

**Comment by nickvollmar** [comment]
@darthShadow – the rollback is manual (there's no "auto-rollback" implemented) :(

**Comment by darthShadow** [comment]
Ok. I am currently using the docker version available @ https://github.com/khawaga/cloudflared which has the latest released version as **2019.09.1** which is broken. 

Downloading the binary manually from the URL in the cloudflare guide(s) has the version **2019.09.2** which doesn't seem to have any corresponding commit here which can be updated in the fork.

Is there any specific commit that I can ask the author of the fork to pull to fix this or should I just stick to the older version for now and wait for an official release from this repo?

**Comment by nickvollmar** [comment]
The commit was made on our internal repo, but we omitted to push it to GitHub. This has been done now. The tag is 2019.9.2 (b3bcce97da05203b49582dc0bbc88118fd0b1f05).
